CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ from `algosdk` and sends it to the network.
 - Added support for new txn opcode fileds `ApprovalProgramPages`, `ClearProgramStatePages`, `NumApprovalProgramPages`, `NumClearProgramStatePages`.
 - Added additional checks for the `maxStackByteElementSize` and `maxProgramLength` in `Runtime`.
 - Added partial support for `bn254` opcodes. The full support delyed until the official release of `TEALv8/v9`.
+- Added a new pre-commit hook to check if there are any `.only`in tests . If there are any it will display error in which files the problem was found.
```

Comment:
```suggestion
- Added a new pre-commit hook to check if there are any `.only`in tests . If there are any it will display error in which files/lines the problem was found.
```